.changeset/stupid-geese-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+"@opennextjs/cloudflare": patch
+---
+
+show error on Next.js versions older than v14
```

Comment:
sorry if it's a silly question, is there a reason why v14 specifically? (I think it's ok but I am just curious if there is something specific that breaks us in <= 13)

Reply:
See https://github.com/opennextjs/opennextjs-cloudflare/pull/234#discussion_r1907320812 and that's something we already discussed with @conico974

Reply:
great, thanks 🙂